### ISSUE TITLE
Does meanValueProperty belong in soccer-common?

### ISSUE THREAD

**Issue by matthew-blackman** [phetsims/center-and-variability#509]
meanValueProperty is currently defined in SoccerSceneModel. Since usage of the mean is specific to Center and Variability, should this be moved to CAVSoccerSceneModel?

**Comment by samreid** [comment]
Yes, I think it should be moved. Please assign me if I should work on that.

**Comment by matthew-blackman** [comment]
@marlitas and I discussed this and agree that moving meanValueProperty to center-and-variability would have some trade-offs. Since both MSaB and CaV make use of the mean, having a central property would help ensure consistency in the logic for how this property is calculated in soccer scenes. We also recognize that comparing one sim to another is a unique situation, and wanted to touch base with @samreid to see how he feels about this line of reasoning.

**Comment by samreid** [comment]
Good point, let's keep it in soccer-common since multiple usage sites need it. Closing.

**Comment by phet-dev** [comment]
Reopening because there is a TODO marked for this issue.
